src/Http/Http.Abstractions/src/StatusCodes.cs

```diff
@@ -283,6 +283,11 @@ public static class StatusCodes
     /// </summary>
     public const int Status451UnavailableForLegalReasons = 451;
 
+    /// <summary>
+    /// HTTP status code 499.
```

Comment:
This is where you'd comment about the origins of the status code.
```suggestion
    /// HTTP status code 499. This is an unofficial status code original defined by NGinx and is commonly used
    /// in logs when the client has disconnected.
```